crates/configuration/src/schema/mod.rs

```diff
@@ -95,6 +95,30 @@ impl From<Type> for ndc_models::Type {
     }
 }
 
+// Should only be used for testing
```

Comment:
Do we have any way of ensuring this for all these?

Reply:
I removed these